src/core/config/type-cast/type-casters/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @prettier
+ */
+
+import booleanTypeCaster from "./boolean"
+import combinedTypeCaster from "./combined"
+import falseTypeCaster from "./false"
+import nullTypeCaster from "./null"
+import numberTypeCaster from "./number"
+import undefinedTypeCaster from "./undefined"
+
+const typeCasters = {
```

Comment:
Aren't we missing `stringTypeCaster`? For example `dom_id` can be defined as nullable string. But now I can define it as nullable number or nullable boolean as well, right?